tests/tests_integration/test_api/test_transformations/test_transformations.py

```diff
@@ -252,6 +254,37 @@ def test_create_instance_edges_type_transformation(self, cognite_client):
 
         cognite_client.transformations.delete(id=ts.id)
 
+    def test_create_instance_data_model_transformation(self, cognite_client):
```

Comment:
Can you rename this test to **test_create_instance_relationship_data_model_transformation**

And create one more **test_create_instance_type_data_model_transformation**

Reply:
test_create_instance_type_data_model_transformation and test_create_instance_relationship_data_model_transformation added